lib/rtoolsHCK.rb

```diff
@@ -144,6 +142,8 @@ def initialize(init_opts)
     @log_to_stdout = init_opts[:log_to_stdout]
     @stdout_logger = Logger.new($stdout) if @log_to_stdout
     @logger = init_opts[:logger]
+    @l_script_file = File.expand_path('../shared/tools/toolsHCK.ps1', __dir__)
```

Comment:
It no longer has to be kept as an instance variable of `RToolsHCK`. We can hardcode it at `Server#deploy_script_file`, which actually uses it.